### ISSUE TITLE
Need Autocorrelate feature for Grafana working

### ISSUE THREAD

**Issue by ambud** [srotya/sidewinder#91]
Use cases are requiring that the auto-correlate feature in grafana be activated so that series that relate to the queried series can automatically be pulled from the database for time series correlation.

**Comment by pauljv92** [comment]
@ambud is this the right issue?

**Comment by ambud** [comment]
@pauljv92 yes this is the one; here's a screenshot of the button in grafana plugin that could trigger the analysis out of other ways of executing it.
![image](https://user-images.githubusercontent.com/9707358/42071639-2af379ec-7b11-11e8-9048-f454e2bc4d3b.png)
